homeassistant/config_entries.py

```diff
@@ -1503,24 +1503,14 @@ def async_shutdown(self) -> None:
                 future.set_result(None)
         self._discovery_event_debouncer.async_shutdown()
 
-    async def async_finish_flow(
+    @callback
+    def async_flow_removed(
         self,
         flow: data_entry_flow.FlowHandler[ConfigFlowContext, ConfigFlowResult],
-        result: ConfigFlowResult,
-    ) -> ConfigFlowResult:
-        """Finish a config flow and add an entry.
-
-        This method is called when a flow step returns FlowResultType.ABORT or
-        FlowResultType.CREATE_ENTRY.
-        """
+    ) -> None:
+        """Handle a removed config flow."""
```

Comment:
The point of this method is to unconditionally remove reauth issues when a reauth flow is removed